db/schema.rb

```diff
@@ -170,6 +194,20 @@
     t.index ["wf_stage"], name: "index_institutions_on_wf_stage"
   end
 
+  create_table "institutions_institutions", id: false, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb3", force: :cascade do |t|
+    t.integer "institution_a_id"
+    t.integer "institution_b_id"
+    t.index ["institution_a_id"], name: "index_institutions_institutions_on_institution_a_id"
+    t.index ["institution_b_id"], name: "index_institutions_institutions_on_institution_b_id"
+  end
+
+  create_table "institutions_to_catalogues", id: false, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb3", force: :cascade do |t|
```

Comment:
What's a catalogue? There is no `catalogue` table, so I'm not sure what table `catalogue_id` is supposed to point to...